js/ui/popup.js

```diff
@@ -668,12 +668,16 @@ var Popup = Overlay.inherit({
         var overlayContent = this.overlayContent().get(0),
             currentHeightStrategyClass = this._chooseHeightStrategy(overlayContent);
 
-        this._setHeightClasses(this.overlayContent(), currentHeightStrategyClass);
         this.$content().css(this._getHeightCssStyles(currentHeightStrategyClass, overlayContent));
+        this._setHeightClasses(this.overlayContent(), currentHeightStrategyClass);
+    },
+
+    _heightStrategyChangeOffset: function(currentHeightStrategyClass, popupHeightParts) {
+        return currentHeightStrategyClass === HEIGHT_STRATEGIES.flex ? -popupHeightParts.popupVerticalPaddings : 0;
```

Comment:
Why not pass `popupVerticalPaddings` as an argument? You do not use other `popupHeightParts`